tests/test_aws.py

```diff
@@ -154,7 +154,7 @@ def test_vm_subnets(project, latest_amzn_image):
         )
 
     # set both
-    with pytest.raises(ValueError):
+    with pytest.raises(ExternalException):  # NOQA F821
```

Comment:
* You forgot to replace another `ValueError` at line 141
* Why is the `# NOQA F821` required?

Reply:
flake8 was acting up :
```bash
flake8 plugins/ tests/
tests/test_aws.py:157:24: F821 undefined name 'ExternalException'
```

Reply:
That's not acting up, that's flake8 telling you you need to import it.

Reply:
Right, of course :+1: